canonical-websites/www.ubuntu.com
# Final sweep: /about

## Done

- Corrected indentation
- Switched divs for sections where appropriate
- Remove -v1 base template and switched include paths

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/about](http://0.0.0.0:8001/about)
- Click through each page and verify that everything is hunky-dory
